tests/test_ag3.py

```diff
@@ -358,6 +358,44 @@ def test_snp_sites(chunks, region):
         assert d.shape == d.compute().shape
 
 
+@pytest.mark.parametrize("chrom", ["2RL", "3RL"])
+def test_snp_sites_for_contig(chrom):
```

Comment:
```suggestion
def test_snp_sites_joined_arms(chrom):
```

Nit, for naming consistency with previous tests.